spec/models/theme_spec.rb

```diff
@@ -1762,5 +1763,24 @@ def included_settings(id)
       expect(theme.find_or_create_owned_color_palette.id).to eq(palette.id)
       expect(theme.owned_color_palette).to eq(palette)
     end
+
+    it "returns system true for Horizon and Foundation themes" do
+      expect(foundation_theme.system?).to be true
+      expect(theme.system?).to be false
+    end
```

Comment:
This is inside the wrong `describe` header. We should probably extract it to `describe #system?`